src/library/autodj/dlgautodj.cpp

```diff
@@ -273,6 +273,13 @@ void DlgAutoDJ::fadeNowButton(bool) {
 void DlgAutoDJ::toggleAutoDJButton(bool enable) {
     AutoDJProcessor::AutoDJError error = m_pAutoDJProcessor->toggleAutoDJ(enable);
     switch (error) {
+        case AutoDJProcessor::ADJ_NOT_TWO_DECKS:
+            QMessageBox::warning(nullptr,
+                tr("Auto DJ"),
+                tr("No deck assigned, Deck must be assigned to enable Auto DJ mode."),
```

Comment:
How about this: 
"Two decks with opposite crossfader orientation are requires to enable Auto DJ mode."

Reply:
or even shorter
"Auto DJ requires two decks assigned to opposite sides of the crossfader."